openapi.yaml

```diff
@@ -16461,6 +16513,61 @@ components:
             label: Enter the DB password
             name: DB_PASSWORD
             example: hunter2
+    StatsDataAvailable:
+      type: object
+      description: >
+        A collection of graph data returned for managed stats.
+      properties:
+        cpu:
+          type: array
+          description: CPU usage stats from the last 24 hours.
+          items:
+            $ref: '#/components/schemas/StatsData'
+        disk:
+          type: array
+          description: Disk usage stats from the last 24 hours.
+          items:
+            $ref: '#/components/schemas/StatsData'
+        swap:
+          type: array
+          description: Swap usage stats from the last 24 hours.
+          items:
+            $ref: '#/components/schemas/StatsData'
+        net_in:
+          type: array
+          description: Inbound network traffic stats from the last 24 hours.
+          items:
+            $ref: '#/components/schemas/StatsData'
+        net_out:
+          type: array
+          description: Outbound network traffic stats from the last 24 hours.
+          items:
+            $ref: '#/components/schemas/StatsData'
+    StatsDataUnavailable:
+      type: array
+      readOnly: true
+      description: >
+        An array of error messages if managed stats are unavaliable.
+      items:
+        type: string
+        example: Graphs are not yet available.
+    StatsData:
+      type: object
+      description: >
+        A stat data point.
+      properties:
+        y:
+          type: integer
+          readOnly: true
+          description: >
+            A stats graph data point.
+            example: 29.94
+        x:
+          type: integer
+          readOnly: true
+          description: >
+            A stats graph data point.
+          example: 11513761600000
```

Comment:
I would switch the order of these. People naturally want to see x before y, regardless of how they are returned in the api.